### ISSUE TITLE
Training on COCO

### ISSUE THREAD

**Issue by dikshant2210** [experiencor/basic-yolo-keras#124]
While training on COCO dataset loss value becomes nan after few iterations.

**Comment by experiencor** [comment]
@dikshant2210 I faced this problem too. Proper warming up or lowering your learning rate will definitely help.

**Comment by dikshant2210** [comment]
I am using adam optimizer so lr is already adaptive. What i found was that there was a exploding gradient problem and using clipnorm and clipvalue I was able to train the model.
